enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -76,20 +84,61 @@ export const MetabaseProviderInternal = ({
   }, [store, config.metabaseInstanceUrl]);
 
   return (
-    <Provider store={store}>
-      <EmotionCacheProvider>
-        <SdkThemeProvider theme={theme}>
-          <AppInitializeController className={className} config={config}>
-            {children}
-          </AppInitializeController>
-        </SdkThemeProvider>
-      </EmotionCacheProvider>
-    </Provider>
+    <EmotionCacheProvider>
+      <SdkThemeProvider theme={theme}>
+        <GlobalFontsStyles baseUrl={config.metabaseInstanceUrl} />
+        <div className={className}>
+          <PortalContainer />
+          {children}
+        </div>
+      </SdkThemeProvider>
+    </EmotionCacheProvider>
   );
 };
 
 export const MetabaseProvider = memo(function MetabaseProvider(
   props: MetabaseProviderProps,
 ) {
-  return <MetabaseProviderInternal store={store} {...props} />;
+  return (
+    <Provider store={store}>
```

Comment:
I did this so we could use useSelector directly in the internal provider, I'm realizing now that it's making unit tests to fail

Reply:
(I fixed the tests)

Reply:
I read the first comment 2 times on different days, and I just got it now, I think this is needed because `useInitData` (which is just added to `MetabaseProviderInternal`) uses Redux under the hood, that's why.

Reply:
Ah, so we lifted the `Provider` outside of the MetabaseProviderInternal for that -- that makes sense.